### ISSUE TITLE
Title in table changes one's to oneś

### ISSUE THREAD

**Issue by crystalfp** [JabRef/jabref#3131]
JabRef 3.8.2
windows 7 6.1 amd64 
Java 1.8.0_144

Load the attached entry in biblatex mode. You will see that in the preview window and in the editor the title correctly show: "one's". Instead in the table the title column shows "oneś". See the attached screenshot.

![bug-title](https://user-images.githubusercontent.com/865866/29483381-45ef9bbc-84a5-11e7-8520-2893ad5477dd.png)
  ```
@Article{Iacoboni2005,
  Title                    = {Grasping the intentions of others with one's own mirror neuron system},
  Author                   = {Iacoboni, Marco and Molnar-Szakacs, Istvan and Gallese, Vittorio and Buccino, Giovanni and Mazziotta, John C and Rizzolatti, Giacomo},
  Date                     = {2005},
  Journaltitle             = {PLoS Biol},
  DOI                      = {10.1371/journal.pbio.0030079},
  Note                     = {(online page: e79)},
  Number                   = {3},
  Pages                    = {529--535},
  Volume                   = {3},
  Abstract                 = {Functional magnetic resonance imaging is used to explore the responses of premotor cortical areas to observing the actions of others.},
  Langid                   = {english}
}
  ```
Thanks for looking!
mario

**Comment by Siedlerchr** [comment]
Thank you for reporting this issue. We think, that is already fixed in the 4.0 beta version and consequently the change will be included in the next release.

We would like to ask you to use a development build from https://builds.jabref.org/master and report back if it works for you.
Be aware that with version 4. the group format has changed, so it is advisable to create a backup of your bib file before.

**Comment by crystalfp** [comment]
OK, solved with: `JabRef-4.0-dev--snapshot--2017-08-19--master--b37b88049.jar`. Thanks!

**Comment by crystalfp** [comment]
OK also using `JabRef-4.0-beta3.jar`
